faq/instances.mdx

```diff
@@ -245,6 +245,17 @@ We provide a wide range of different Linux distributions and InstantApps for Ins
 * CentOS 8
 * Fedora 32
 
+### What are the differences between ENT1 and POP2 Instances?
+
+Both ENT1 and POP2 Instance types share the following features:
+- Identical hardware specifications
+- Dedicated vCPU allocation
+- Same pricing structure
+- Accelerated booting process
+
+POP2 Instances provide CPU- and memory-optimized variants tailored to suit your workload requirements more effectively. The primary distinction between ENT1 and POP2 lies in SEV, which is disabled for POP2 Instances.
```

Comment:
Maybe an explanation of what SEV is, or a link?

Reply:
```suggestion
POP2 Instances provide CPU- and memory-optimized variants tailored to suit your workload requirements more effectively. The primary distinction between ENT1 and POP2 lies in [AMD Secure Encrypted Virtualization (SEV)](https://www.amd.com/fr/developer/sev.html), which is disabled for POP2 Instances.
```